index.html

```diff
@@ -163,67 +189,116 @@ <h2>Profesional Art <br>Printing Data</h2>
                <li><button class="projects-btn" type="button">Ruby</button></li>
            </ul>
        </div>
-    </div>  
+    </div> 
+    </div>
+     
                          </div>
                         <button class="see-project">See project</button>
-                    </div>
-                </div>
-            </div>
+                    
+                    </div>       
         </div>
+               </div>
+            </div>
         </div>
-       </div>       
+        
+       </div>
```

Comment:
try to improve on unecessarry indentations, your code looks good though

Reply:
- [ ] Good work giving credit where it's due.  I will point out a few things below that could have improved this. 👍🏾 
 - Please remember to be polite. This makes it easy for the students to receive and implement your reviews without feeling ordered. An example would be "**Please avoid unnecessary indentations**" 😸 
 - Once you have offered your suggestion, please give a reason why you have offered the suggestion. You could say "**Unnecessary indentations make it hard for other people to read your code**" ✍🏾 
 - Whenever possible, You could offer suggestions on how to improve the highlighted issue. For example "**I think you should use HTML  linters to help enforce proper code indentation in this project**" 🤝